sample/build.gradle

```diff
@@ -18,6 +18,9 @@ android {
     sourceSets {
         main.java.srcDirs += "src/main/kotlin"
     }
+    lintOptions {
+        abortOnError false
```

Comment:
somehow the error occurs only on GitHub Action, not on my local 🤔 

https://github.com/permissions-dispatcher/PermissionsDispatcher/runs/2008862691?check_suite_focus=true#step:6:569